Sources/TensorFlow/Core/TensorGroup.swift

```diff
@@ -38,6 +38,42 @@ public protocol TensorArrayProtocol {
     init<C: RandomAccessCollection>(_handles: C) where C.Element: _AnyTensorHandle
 }
 
+extension TensorArrayProtocol {
+    init<C: RandomAccessCollection>(_handles: C) where C.Element: _AnyTensorHandle {
+        let status = TF_NewStatus()
+        defer { TF_DeleteStatus(status) }
+        let buffer: UnsafeMutablePointer<CTensorHandle> =
+            UnsafeMutablePointer.allocate(capacity: _handles.count)
```

Comment:
```suggestion
            UnsafeMutablePointer<CTensorHandle>.allocate(capacity: _handles.count)
```
This can be moved to the end of the previous line once the type signature gets removed.